ui/ui-components/components/record/List.tsx

```diff
@@ -41,10 +45,10 @@ export default function RecordsList(props) {
   const limit = 100;
   const { offset, setOffset } = useOffset();
   const [searchKey, setSearchKey] = useState<string>(
-    router.query?.searchKey || props.searchKey || ""
+    (router.query?.searchKey as string) || ""
```

Comment:
Now that this is typed it showed that searchKey was never provided.

Reply:
Looks like it's being passed in the property records page, for ui-enterprise

https://github.com/grouparoo/grouparoo/blob/main/ui/ui-enterprise/pages/model/%5BmodelId%5D/source/%5BsourceId%5D/property/%5BpropertyId%5D/records.tsx#L47

Reply:
Oh weird, that didn't come up when I looked for references. I'll fix.